src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala

```diff
@@ -89,8 +89,8 @@ class SequenceRowsRelation(config: RelationConfig, sequenceId: CogniteId)(val sq
         }
       val projectedRows =
         client.sequenceRows
-          .queryById(
-            sequenceInfo.id,
+          .query(
+            CogniteInternalId(sequenceInfo.id),
```

Comment:
I think you can just use `sequenceId` from `class SequenceRowsRelation(config: RelationConfig, sequenceId: CogniteId)`?

Reply:
right, we get the `sequenceInfo` by using that `sequenceId` any way :)